src/list_ordered_multimap.rs

```diff
@@ -4084,6 +4166,27 @@ mod test {
         assert_eq!(iter.next(), None);
     }
 
+    #[test]
+    fn test_iter_size_hint() {
```

Comment:
Move below test_iter_mut_debug for alphabetical order

Reply:
I moved it below `test_iter_mut_size_hint`. I hope you meant this. I don't see it alphabetically otherwise.